### ISSUE TITLE
chefspec tests broken

### ISSUE THREAD

**Issue by TD-4242** [chef-cookbooks/ohai#34]
Started getting this after the 3.0.0 release in any of our internal wrapper cookbooks that include ohai in the dependency tree.

```
================================================================================
Recipe Compile Error in /tmp/d20160314-459-1k4mp83/cookbooks/ohai/libraries/matchers.rb
================================================================================

NoMethodError
-------------
undefined method `define_runner_method' for ChefSpec:Module

Cookbook Trace:
---------------
  /tmp/d20160314-459-1k4mp83/cookbooks/ohai/libraries/matchers.rb:4:in `<top (required)>'

Relevant File Content:
----------------------
/tmp/d20160314-459-1k4mp83/cookbooks/ohai/libraries/matchers.rb:

  1:  # encoding: utf-8
  2:
  3:  if defined?(ChefSpec)
  4>>   ChefSpec.define_runner_method(:ohai_hint)
  5:
  6:    def create_ohai_hint(resource)
  7:      ChefSpec::Matchers::ResourceMatcher.new(:ohai_hint, :create, resource)
  8:    end
  9:
 10:    def delete_ohai_hint(resource)
 11:      ChefSpec::Matchers::ResourceMatcher.new(:ohai_hint, :delete, resource)
 12:    end
 13:  end

```

on chefdk 0.11.2

Thanks,
-Bill

**Comment by tas50** [comment]
@TD-4242 I released 3.0.1, which should fix this.  let me know if it's still a problem.

**Comment by TD-4242** [comment]
looks like it did it.  Thanks!!
